data_steward/analytics/cdr_ops/controlled_tier_qc/sql/query_templates.py

```diff
@@ -116,6 +116,8 @@
     SELECT {{ new_id }}
     {% if mapping_table == 'site_maskings' %}
     FROM `{{ project_id }}.{{ pipeline_dataset }}.{{ mapping_table }}`
+    {% elif mapping_table == '_deid_questionnaire_response_map' %}
+    FROM `{{ project_id }}.{{ questionnaire_response_dataset }}.{{ mapping_table }}`
```

Comment:
The dataset containing the table `_questionnaire_response_map`  is given as a notebook parameter. This check should use the dataset defined by the parameter instead of falling to the else clause and therefore the 'mapping_dataset'.